receipt_parser_core/receipt.py

```diff
@@ -108,11 +108,14 @@ def parse_items(self):
         items = []
         item = namedtuple("item", ("article", "sum"))
 
-        ignored_words = self.config.sum_keys
-        for word in self.config.ignore_keys:
-            ignored_words.append(word)
+        ignored_words = self.config.ignore_keys
+        stop_words = self.config.sum_keys
 
         for line in self.lines:
+            for stopWord in stop_words:
+                if fnmatch.fnmatch(line, f"*{stopWord}*"):
+                    return items
```

Comment:
I'm confused; wouldn't this stop parsing *all* items if one line contains a stop word? 🤔

Reply:
Yes, parsing should stop at a stopword, as the regex maybe matches some more wrong articles after "Summe" or "Rückgeld"

Reply:
Ah gotcha; thanks!